src/main.rs

```diff
@@ -70,12 +70,21 @@ fn sse(req: &HttpRequest<SSEClientState>) -> HttpResponse {
 
     let r = HttpResponse::build(StatusCode::OK)
         .content_type("text/event-stream")
+        .content_encoding(http::header::ContentEncoding::Identity)
+        .no_chunking()
+        .force_close()
```

Comment:
This works fine for me with just the `.content_encoding` part, no need for `connection: close`. Care to change that, and I'll merge?